front/src/applications/operationalStudies/components/SimulationResults/Allowances/StandardAllowanceDefault.tsx

```diff
@@ -45,7 +45,7 @@ interface StandardAllowanceDefaultProps {
     value: number;
   };
   getAllowanceTypes?: (typeKey: string) => AllowanceType[];
-  isStdcm?: boolean;
+  isCondensed?: boolean;
```

Comment:
I think this is better :) What do you think @Math-R ?
```suggestion
  isCondensed: boolean = false;
```